github/github.go

```diff
@@ -102,6 +102,9 @@ const (
 
 	// https://developer.github.com/changes/2017-07-26-team-review-request-thor-preview/
 	mediaTypeTeamReviewPreview = "application/vnd.github.thor-preview+json"
+
+	// https://developer.github.com/changes/2017-09-21-additional-endpoints-for-github-apps/
+	mediaTypeGitHubAppsPreview = "application/vnd.github.machine-man-preview+json"
```

Comment:
I just noticed we already have the same custom media type constant defined above:

```Go
// https://developer.github.com/changes/2016-09-14-Integrations-Early-Access/
mediaTypeIntegrationPreview = "application/vnd.github.machine-man-preview+json"
```